packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,78 @@
+.spinner {
+  position: relative;
+  pointer-events: none;
+  opacity: 0.6;
+}
+
+.spinner:after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: var(--space-larger);
+  height: var(--space-larger);
+  box-shadow: var(--shadow-dark);
+  margin: -1.125rem 0 0 -1.125rem;
+  border: 0.375rem solid rgba(1, 27, 37, 1);
+  border-top-color: rgba(255, 255, 255, 1);
+  border-radius: 50%;
+  background-color: rgba(1, 27, 37, 0.3);
+  animation-duration: 1s;
+  animation-iteration-count: infinite;
+  animation-name: animation-spin;
+  animation-timing-function: linear;
+  will-change: transform;
+}
+
+@-webkit-keyframes animation-spin {
+  0% {
+    -webkit-transform: rotate(0deg);
```

Comment:
Question for @seanhealy maybe, but shouldn't our postcss handle prefixes?

Reply:
It should / does.

Reply:
@eddysims could you briefly explain postcss and prefixes?? 🤔Shot in the dark, but I'm guessing that this first 0% keyframe is unnecessary?